apps/builder/app/shared/sync/sync-stores.ts

```diff
@@ -98,18 +96,10 @@ export const createObjectPool = () => {
     new NanostoresSyncObject("dataSourceVariables", $dataSourceVariables),
     new NanostoresSyncObject("resourceValues", $resourceValues),
     new NanostoresSyncObject("selectedPageHash", $selectedPageHash),
-    new NanostoresSyncObject(
-      "selectedInstanceBrowserStyle",
-      $selectedInstanceBrowserStyle
-    ),
     new NanostoresSyncObject(
       "selectedInstanceIntanceToTag",
```

Comment:
selectedInstanceIntanceToTag no longer synced from canvas, can be removed too

Reply:
Should be synced as selectedInstanceSelector can be set from canvas

Reply:
Ah, actually synced when selectInstance is invoked from canvas. Nevermind